### PR TITLE
Wrote the code for a double-FOV composite stream.

### PR DESCRIPTION
**Pull request by brogan20**
The code is in a separate Vision.java class, so if it needs to go somewhere else then please move it where it's needed

### CODE REVIEW

File: src/org/usfirst/frc/team3695/robot/Vision.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            //The arraylist of left and right sources is needed for concatenating
+            ArrayList<Mat> sources = new ArrayList<>();
+            sources.add(leftSource);
+            sources.add(rightSource);
+
+            //Concatenation of both matrices
+            Mat concat = new Mat();
+
+            //Puts the combined video on the SmartDashboard (I think)
+            CvSource outputStream = CameraServer.getInstance().putVideo("Concat", 3840, 1080);
```
Comment (wh1ter0se) [inline src/org/usfirst/frc/team3695/robot/Vision.java]: where did 3840 and 1080 come from? are these constants?

**Comment by brogan20** [comment]
Should be set now.

**Comment by wh1ter0se** [comment]
WOAH THERE. MASTER BRANCH. AHHHHHHHHH.

**Comment by brogan20** [comment]
Going to reopen to pull to develop

**Pull request closed without merging**